tests/openapi-administration.json

```diff
@@ -2573,6 +2573,201 @@
+                        "schema": {
+                            "type": "string",
+                            "enum": [
+                                "v2"
+                            ],
+                            "default": "v2"
+                        }
+                    },
+                    {
+                        "name": "OCS-APIRequest",
+                        "in": "header",
+                        "description": "Required to be true for the API request to pass",
+                        "required": true,
+                        "schema": {
+                            "type": "boolean",
+                            "default": true
+                        }
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "object",
+                                    "required": [
+                                        "ocs"
+                                    ],
+                                    "properties": {
+                                        "ocs": {
+                                            "type": "object",
+                                            "required": [
+                                                "meta",
+                                                "data"
+                                            ],
+                                            "properties": {
+                                                "meta": {
+                                                    "$ref": "#/components/schemas/OCSMeta"
+                                                },
+                                                "data": {
+                                                    "oneOf": [
+                                                        {
+                                                            "type": "string"
+                                                        },
+                                                        {
+                                                            "type": "integer",
+                                                            "format": "int64"
+                                                        },
+                                                        {
+                                                            "type": "number",
+                                                            "format": "float"
+                                                        },
+                                                        {
+                                                            "type": "number",
+                                                            "format": "double"
+                                                        },
```

Comment:
How is this going to be `oneOf`?
Wouldn't a `0.0` match both float and double?

Reply:
I thought about this as well. Maybe the format should not be considered to distinguish the schemas.

Reply:
Same for int32/int64 formats. I'll remove the format from the comparison because it is not safe.

Reply:
I'm new to the codebase so correct me if I'm wrong but shouldn't this be covered by a test?

I also had another read of the jsonschema documentation and aparently `2.0` is also a valid integer.
https://json-schema.org/understanding-json-schema/reference/numeric

I'm not sure how relevant this information is regarding php and would prfer having them typed as double/float :)

Reply:
True, I'll add a test.

Those integers will never appear in our specs because they will always be double or float.

Reply:
But this means `oneof: [double | integer]` is also wrong as double is a superset of an integer.

Reply:
Because any integer is also a valid double? Damn I hate JSON schema :joy: